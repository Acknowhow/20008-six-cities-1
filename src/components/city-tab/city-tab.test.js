import React from 'react';
import renderer from 'react-test-renderer';
import CityTab from './city-tab.jsx';

it(`CityTab renders correctly after launch`, () => {

  const tree = renderer
    .create(<CityTab
      renderTab={jest.fn()}
    />, {
      createNodeMock: () => {
        return {};
      }})
    .toJSON();

  expect(tree).toMatchSnapshot();
});
