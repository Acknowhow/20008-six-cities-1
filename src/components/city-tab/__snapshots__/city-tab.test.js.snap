// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CityTab renders correctly after launch 1`] = `null`;
